custom/openclinica/const.py

```diff
@@ -1,3 +1,6 @@
+# Only for the KEMRI study. This whole file is dropped for generic study apps
+STUDY_APPS = ('Captopril Study',)
```

Comment:
if they change the app name will this break? seems a bit brittle - could you use the ID instead?